aries-site/src/layouts/content/Example/ExampleResources.js

```diff
@@ -122,6 +123,11 @@ export const ExampleResources = ({
           </SyntaxHighlighter>
         </CollapsibleSection>
       )}
+      {caption && (
+        <Paragraph size="small" margin="none">
+          {caption}
+        </Paragraph>
```

Comment:
Also, for some reason the example with caption isn't aligning with the example below it:

<img width="697" alt="Screen Shot 2021-11-02 at 11 38 00 AM" src="https://user-images.githubusercontent.com/12522275/139925098-e22fcd17-66e0-40d1-8624-0774791b778e.png">

Reply:
I think the 2 above are indented so their smaller I can look into the width